### PR TITLE
Prettier. Would like to merge it with develop branch

### PR DESCRIPTION
**Pull request by AndriiBabushko**
Add prettier dependency.

### CODE REVIEW

File: package.json
Comment (FullmetalBober) [inline package.json:1]: 'prettier' dependency should be saved as devDependencies

File: .idea/dbnavigator.xml
Comment (FullmetalBober) [inline .idea/dbnavigator.xml:1]: Don't commit fucking idea folder

File: .gitignore
Comment (FullmetalBober) [inline .gitignore:1]: Why you added to gitignore so many .env files?
The only file that cannot be committed is .env*.local

Comment (AndriiBabushko) [reply .gitignore:1]: Fixed

Comment (FullmetalBober) [reply .gitignore:1]: _*_

File: .idea/dbnavigator.xml
Comment (FullmetalBober) [reply .idea/dbnavigator.xml:1]: Why idea is there?

Comment (AndriiBabushko) [reply .idea/dbnavigator.xml:1]: Fixed

File: .gitignore
Comment (AndriiBabushko) [reply .gitignore:1]: So its done or not?

Comment (FullmetalBober) [reply .gitignore:1]: Done

**Pull request merged**